tests/command_line/test_gulp_caller.py

```diff
@@ -132,15 +131,6 @@ def test_structure_lines_no_frac_coords(self):
         assert "cell" not in inp_str
         assert "cart" in inp_str
 
-    @unittest.skip("Not Implemented yet")
-    def test_specie_potential(self):
-        pass
-
-    @unittest.expectedFailure
-    def test_library_line_explicit_path(self):
```

Comment:
I would remove this test as it is hard-coded to a specific path, even with `expectedFailure` it doesn't really seem to be checking anything